packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -185,3 +186,7 @@ function isSOQLExtensionInstalled() {
   const soqlExtension = vscode.extensions.getExtension(soqlExtensionName);
   return soqlExtension !== undefined;
 }
+
+export const enableSyncInitJobs: boolean = vscode.workspace
```

Comment:
This logic should go in [settings.ts](https://github.com/forcedotcom/salesforcedx-vscode/blob/399189a6ea09ef9e963f152fe0011509d8d64d77/packages/salesforcedx-vscode-apex/src/settings.ts#L11)

The languageServer shouldn't be in the business of reading vscode settings.